### PR TITLE
update suite.submit script and add new controls and cice.setup option…

### PR DESCRIPTION
**Pull request by apcraig**
## PR checklist
- [X] Short (1 sentence) summary of your PR: 
    Update suite.submit script and add new features.
- [X] Developer(s): 
    apcraig
- [X] Suggest PR reviewers from list in the column to the right.
- [X] Please copy the PR test results link or provide a summary of testing completed below.
    Tested on izumi, defaults settings work fine and are backward compatible.
    Tested the suite.submit manually as well.
- How much do the PR code changes differ from the unmodified code? 
    - [X] bit for bit
    - [ ] different at roundoff level
    - [ ] more substantial 
- Does this PR create or have dependencies on Icepack or any other models?
    - [ ] Yes
    - [X] No
- Does this PR add any new test cases?
    - [ ] Yes
    - [X] No
- Is the documentation being updated? ("Documentation" includes information on the wiki or in the .rst files from doc/source/, which are used to create the online technical docs at https://readthedocs.org/projects/cice-consortium-cice/. A test build of the technical docs will be performed as part of the PR testing.)
    - [X] Yes
    - [ ] No, does the documentation need to be updated at a later time?
        - [ ] Yes
        - [ ] No 
- [ ] Please provide any additional information or relevant details below:

Added cice.setup options

-   --setup-only
-   --build-only
-   --run-interactive

Added ability to set env variables for use with suite.submit directory

-   SUITE_BUILD
-   SUITE_RUN
-   SUITE_SUBMIT

Removed suite.run script, this is now part of suite.submit script.

Also updated the documentation.  This is an update of #357 based on the latest code and discussions in that PR.

### CODE REVIEW

**Comment by apcraig** [comment]
I am closing #357.

**Comment by apcraig** [comment]
I'm open about how to name the options.  I think build-only implicitly means you have to create the cases, but understand these things can be tricky.  Also, maybe we need 4 options.  For clarity, how about

  - setup-only
  - setup-and-build-only
  - setup-and-build-and-run-interactive
  - setup-and-build-and-submit

That seems a little too wordy but is clear.  Other proposals?

**Comment by eclare108213** [comment]
How about 
 * setup
 * setup-build
 * setup-build-run
 * setup-build-submit

**Comment by apcraig** [comment]
How about 

 - setup-only
 - setup-build
 - setup-build-run
 - setup-build-submit

**Comment by apcraig** [comment]
I updated the options and committed.  Can still make another change if needed.

**Comment by phil-blain** [comment]
I'm still looking at the tutorial activities and will have more time to test this next week...

**Comment by apcraig** [comment]
Just a quick update that we are waiting for @phil-blain to do a test/review.  There is no rush or urgency, but that's why this PR is on hold for the moment.

**Comment by apcraig** [comment]
I just made some commits to update the documentation as suggested by @phil-blain.  I think this improves things and have confirmed the formatting in the RTD output.  

With regard to the csh vs bash syntax, my preference is to avoid providing multiple examples for multiple shells if it's something pretty basic, so instead, I have added a quick comment about that.  I think for the conda stuff, given the install is different in different shells, then it should be documented.  For this, where we're just setting an env variable, I think we should be able to rely on the community to understand that difference.

I think we can probably merge before the weekend testing unless there are other comments or feedback.

**Comment by phil-blain** [comment]
looks good to me!
